packages/react-native-reanimated/Common/cpp/reanimated/NativeModules/ReanimatedModuleProxy.cpp

```diff
@@ -918,18 +900,11 @@ void ReanimatedModuleProxy::performOperations() {
 }
 
 void ReanimatedModuleProxy::requestFlushRegistry() {
-  jsi::Runtime &rt =
-      workletsModuleProxy_->getUIWorkletRuntime()->getJSIRuntime();
-  requestRender_(
-      [weakThis = weak_from_this()](double time) {
-        auto strongThis = weakThis.lock();
-        if (!strongThis) {
-          return;
-        }
-
-        strongThis->shouldFlushRegistry_ = true;
-      },
-      rt);
+  requestRender_([weakThis = weak_from_this()](double time) {
```

Comment:
This is a nit but since we're touching this part of the code, we can also unify the argument name with other occurrences.
```suggestion
  requestRender_([weakThis = weak_from_this()](const double timestamp) {
```